ros_buildfarm/common.py

```diff
@@ -10,9 +10,7 @@ class JobValidationError(Exception):
     if validation fails, e.g. because the requested package
     is not available in the specified index.yaml
     """
-
-    def __init__(self, message):
-        self.message = message
+    pass
```

Comment:
Have you tried that in the case of such an exception the message is still shown as before?